src/opf/modules.py

```diff
@@ -260,12 +278,13 @@ def parse_bus(self, bus: torch.Tensor):
         # Convert voltage and power to per unit
         vr = bus[:, 0, :]
         vi = bus[:, 1, :]
-        pg = bus[:, 2, :]
-        qg = bus[:, 3, :]
+        # pg = bus[:, 2, :]
+        # qg = bus[:, 3, :]
```

Comment:
Either rename to `parse_output` and process (bus,gen) or create two functions `parse_bus` and `parse_gen`.